packages/babel-preset-default/index.js

```diff
@@ -6,18 +6,7 @@ module.exports = {
 		! isTestEnv && [ require( 'babel-preset-env' ), {
 			modules: false,
 			targets: {
-				browsers: [
-					'last 2 Chrome versions',
-					'last 2 Firefox versions',
-					'last 2 Safari versions',
-					'last 2 Edge versions',
-					'last 2 Opera versions',
-					'last 2 iOS versions',
-					'last 1 Android version',
-					'last 1 ChromeAndroid version',
-					'ie 11',
-					'> 1%',
-				],
+				browsers: ['extends @wordpress/browserslist-config'],
```

Comment:
Standards: Needs space within the square brackets:

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#arrays-and-function-calls

Noting that this could serve as interesting evidence against the whitespace exceptions for array and object keys, since it's hard to distinguish between them and arrays proper.

Reply:
We don't have Eslint configured for this repository :(